proxy/common_neon/erc20_wrapper.py

```diff
@@ -155,7 +155,7 @@ def create_claim_to_ix(self, owner: SolPubKey,
             to_acct.address,
             amount
         ).build_transaction(
-            {'nonce': nonce, 'gasPrice': 0}
+            {'nonce': nonce, 'gasPrice': 0, 'from': signer_acct.address}
```

Comment:
`from` property doesn't exist in the transaction structure.
the proxy recovers it from the signature.
it is the surplus modification.